src/HelpScout/ApiClient.php

```diff
@@ -637,54 +637,105 @@ private function getItem($url, $params, $method, $model) {
 	}
 
 	/**
-	 * @param  integer $statusCode The HTTP status code returned
-	 * @param  string  $type       The type of request (e.g., GET, POST, etc.)
-	 * @param  integer $expected   The expected HTTP status code
+	 * @param  integer             $statusCode   The HTTP status code returned
+	 * @param  string              $type         The type of request (e.g., GET, POST, etc.)
+	 * @param  integer             $expected     The expected HTTP status code
```

Comment:
I believe that docs here could be `integer|array` to allow a collection of expected status codes.